src/components/Login.js

```diff
@@ -40,22 +40,18 @@ const themeColors = {
   4: ['#FF94DB', 'white'],
 };
 
-class Login extends React.Component {
-  state = {
-    dummy: false,
-    index: Math.floor(Math.random() * 5),
-  };
+const Login = ({ create, initialState }) => {
+  const [dummy, setDummy] = useState(false);
+  const [index, setIndex] = useState(Math.floor(Math.random() * 5));
 
   // basically, when the window resizes, we should recalculate get SVG - the window parameters change!
-  componentDidMount = () => {
-    window.addEventListener('resize', () => this.setState({ dummy: !this.state.dummy }));
-  };
+  useEffect(() => {
+    window.addEventListener('resize', () => setDummy((_dummy) => !_dummy));
 
-  componentWillUnmount = () => {
-    window.removeEventListener('resize', () => this.setState({ dummy: !this.state.dummy }));
-  };
+    return () => window.removeEventListener('resize', () => setDummy((_dummy) => !_dummy));
+  }, [dummy]);
```

Comment:
Don't think this needs to be dependent on `dummy`

Reply:
Copy that.  If not dependent on dummy, the dummy var is unused throughout the file.  Should I just set up lint-ignore to silence the warning that dummy is not used?

Reply:
Just to jump in here - I think the original intent of the code was to force a component re-render. If you can create a custom hook and/or some sort of listener that triggers a re-render (and then gets the window size, etc.), you can remove the dependency on the `dummy` state variable.

Reply:
Here's one common hook used for the same case: https://usehooks.com/useWindowSize/

Here's an interesting SO post on something similar; you may (?) have to use `useRef`: https://stackoverflow.com/questions/66588340/custom-hook-for-window-resize